src/paths.ts

```diff
@@ -18,10 +19,16 @@ interface VSCMakePaths {
 
 class WindowsEnvironment {
     get AppData(): string | undefined {
+        if (process.env['CMT_TESTING'] === '1') {
+            return path.join(vscode.workspace.workspaceFolders![0].uri.fsPath, '.vscode');
+        }
         return process.env['APPDATA'];
     }
 
     get LocalAppData(): string | undefined {
+        if (process.env['CMT_TESTING'] === '1') {
+            return path.join(vscode.workspace.workspaceFolders![0].uri.fsPath, '.vscode');
```

Comment:
Might be nice to still have a separate subfolder like `.vscode/LocalAppData` to avoid polluting the .vscode folder (if tests ever write to %TEMP%)

Reply:
The extension already creates a folder in `%LocalAppData%` called `CMakeTools`.  This is just for tests so it just gets deleted at the end of CI.  Though I should add that path to the `.gitignore`